bin/word-tokenization-benchmark

```diff
@@ -21,7 +21,7 @@ parser.add_argument(
 parser.add_argument(
     "--test-file",
     action="store",
-    help="path to test file"
+    help="path to test file, i.e. ground truth"
```

Comment:
Can you add type file?

Reply:
Both input and test files are plain text files.